index.d.ts

```diff
@@ -19,6 +19,15 @@ declare namespace tempy {
 			readonly name?: string;
 		}
 	>;
+
+	type DirectoryOptions = {
+		/**
+		_You usually won't need this option. Specify it only when actually needed._
+
+		Directory prefix.
```

Comment:
The doc comment should also elaborate on the use-cases for this option.

Reply:
Fixed!